src/extension.ts

```diff
@@ -148,6 +150,23 @@ export async function activate(context: vscode.ExtensionContext) {
             () => { vscode.env.openExternal(vscode.Uri.parse(githubUrl)) }
         )
 
+        vscode.commands.registerCommand(
+            'aws.welcome',
+            async () => {
+                const view = vscode.window.createWebviewPanel(
+                    'html',
+                    localize('AWS.command.welcome.title', 'AWS Toolkit - Welcome'),
+                    vscode.ViewColumn.Active
+                )
+
+                const fileText = convertPathTokensToPath(
```

Comment:
Consider producing `fileText` prior to creating `view` in case an error occurs.

Reply:
✔️